frontend/src/pages/Dashboard/SessionDashboard.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { FunctionComponent } from "react";
+import styled from "styled-components";
+import { SessionStats } from "../../models/types";
+import { CLOUD, INK } from "../../constants/colors";
+
+interface SessionDashboardParams {
+  userSessionData: SessionStats;
+}
+
+interface StatParams {
+  stat: number;
+  description: string;
+}
+
+const StatContainer = styled.ul`
+  padding: 24px;
+`;
+
+const StatTitle = styled.p`
+  color: #000000;
+  font-family: "Rubik";
+  font-size: 26px;
+  font-weight: 700;
+`;
+
+const FormattedStat = styled.li`
+  font-family: "Rubik";
+  font-size: 20px;
+  font-weight: 700;
+  text-transform: lowercase;
+  background: ${CLOUD};
+  list-style: none;
+  border-radius: 12px;
+  padding: 24px;
+  margin: 12px 0px;
+  display: flex;
+  justify-content: space-between;
+  @media (max-width: 900px) {
+    display: block;
+  }
+`;
+
+const StatNumber = styled.p`
+  color: ${INK};
+  margin: 0px;
+  padding: 0 0 0 0.5em;
+  @media (max-width: 900px) {
+    padding: 0;
+  }
+`;
+
+const StatDescription = styled.p`
+  color: #000000;
+  margin: 0px;
+`;
+
+const Stat: FunctionComponent<StatParams> = ({
+  stat,
+  description,
+}) => {
+  return (
+    <FormattedStat>
+      <StatDescription>{description}</StatDescription>
+      <StatNumber>{formatDuration(stat)}</StatNumber>
```

Comment:
For words answered correctly/incorrectly, We should not format it to be a minute/second style, but just keep the normal values.